assets/lokomotive-kubernetes/bootkube/resources/charts/lokomotive/Chart.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v2
+name: lokomotive
+description: A Helm chart providing Lokomotive-specific system workloads like admission webhook server.
+type: application
+
+version: 0.1.0
+
+appVersion: 0.3.0
```

Comment:
If we don't use `appVersion` for image tag, maybe we can remove it? I guess we will be pinning image tag to the specific commit, which we cannot put in `appVersion` field.